ServiceAPIExtensions/Controllers/ContentAPiController.cs

```diff
@@ -65,7 +65,7 @@ ContentReference FindContentReference(string reference)
             return ContentReference.EmptyReference;
         }
 
-        static Dictionary<string, object> MapContent(IContent content, int recurseContentLevelsRemaining)
+        Dictionary<string, object> MapContent(IContent content, int recurseContentLevelsRemaining)

```

Comment:
I'd like to keep this `static`, such that we have _some_ guarantee (and at least indicate the intention) that this method doesn't cause side effects or database calls. We can use `_typerepo.List().ToDictionary(x => x.ID, x => x)` to pass in a map of ContentTypeID's to the ContentType. I don't expect (this we might need to check) a lot of ContentTypes in the system and also this data is probably cached.